script/release/DSL-scripts/KIE_pushReleaseBranches.sh

```diff
@@ -47,13 +47,16 @@ fi
 
 # update kie-parent-metadata
 cd droolsjbpm-build-bootstrap/
-
 # change <version.org.uberfire>, <version.org.dashbuilder> and <version.org.jboss.errai>
 sed -i "$!N;s/<version.org.uberfire>.*.<\/version.org.uberfire>/<version.org.uberfire>$UBERFIRE_VERSION<\/version.org.uberfire>/;P;D" pom.xml
 sed -i "$!N;s/<version.org.dashbuilder>.*.<\/version.org.dashbuilder>/<version.org.dashbuilder>$DASHBUILDER_VERSION<\/version.org.dashbuilder>/;P;D" pom.xml
 sed -i "$!N;s/<version.org.jboss.errai>.*.<\/version.org.jboss.errai>/<version.org.jboss.errai>$ERRAI_VERSION<\/version.org.jboss.errai>/;P;D" pom.xml
+sed -i "$!N;s/<latestReleasedVersionFromThisBranch>.*.<\/latestReleasedVersionFromThisBranch>/<latestReleasedVersionFromThisBranch>$RELEASE_VERSION<\/latestReleasedVersionFromThisBranch>/;P;D" pom.xml
 
-cd ..
+cd ../jbpm
+# change release.version in jbpm/bpm-installer/src/main/resources/build.properties
+sed -i "$!N;s/release.version=.*./release.version=$RELEASE_VERSION/;P;D" jbpm-installer/src/main/resources/build.properties
```

Comment:
This should not be necessary. The maven module itself needs to update the version accordingly during the build. Or am I missing something here?